cmd/internal/cli/remote.go

```diff
@@ -454,56 +448,24 @@ var RemoteStatusCmd = &cobra.Command{
 	DisableFlagsInUseLine: true,
 }
 
-// RemoteAddKeyserverCmd apptainer remote add-keyserver [option] [remoteName] <keyserver_url>
+// RemoteAddKeyserverCmd apptainer remote add-keyserver (deprecated)
 var RemoteAddKeyserverCmd = &cobra.Command{
-	Args:   cobra.RangeArgs(1, 2),
-	PreRun: setKeyserver,
+	Args: cobra.RangeArgs(0, 2),
 	Run: func(cmd *cobra.Command, args []string) {
-		uri := args[0]
-		name := ""
-		if len(args) > 1 {
-			name = args[0]
-			uri = args[1]
-		}
-
-		if cmd.Flag(remoteKeyserverOrderFlag.Name).Changed && remoteKeyserverOrder == 0 {
-			sylog.Fatalf("order must be > 0")
-		}
-
-		if err := apptainer.RemoteAddKeyserver(name, uri, remoteKeyserverOrder, remoteKeyserverInsecure); err != nil {
-			sylog.Fatalf("%s", err)
-		}
+		sylog.Fatalf("command name changed: use 'keyserver add' instead of 'remote add-keyserver'")
```

Comment:
If we are going to leave these command here what is the point of removing the functions and making them error.  Seems like a better idea to leave them in a working state and just add a deprecation warning.  At least until the next minor release (when behavior can change).

Reply:
It was already going to go into the next minor release, not a patch release.  Are you saying it ought to be two minor releases before it gets removed?

Reply:
I feel like there ought to be at least one release in which there is a deprecation warning before the behavior changes.  If the next release is a minor one then yes imho this should probably be deprecated with a warning for the entire patch release cycle.

Reply:
Ok I made that change.